### PR TITLE
Fixed size of image upload container

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/1428

#### What's this PR do?
Fixed image upload error message position and size of image upload container

#### Where should the reviewer start?
see static/js/components/ProfileImageUploader.js

#### How should this be manually tested?
- open user profile and click user pic, a dialog will appear

@pdpinch @aliceriot @noisecapella 

#### Screenshots (if appropriate)
screen shot 2016-10-20 at 4 46 29 pm

### CODE REVIEW

File: static/scss/profile-image-uploader.scss [hunk 1]
```diff
@@ -22,6 +22,6 @@
 
   .photo-active-item {
     width: 100%;
-    height: 500px;
+    height: 300px;
```
Comment (pdpinch) [inline static/scss/profile-image-uploader.scss]: this seems like a better default value, but can we be smarter here? Does it make sense to use a percentage? or a calculation based on the viewport height minus the chrome?

Comment (aliceriot) [inline static/scss/profile-image-uploader.scss]: I support that plan! optimally we'd show it at the size it currently is on a big display, and shrink down only when needed on smaller displays

Comment (amir-qayyum-khan) [inline static/scss/profile-image-uploader.scss]: Done @pdpinch @aliceriot

**Comment by aliceriot** [comment]
If you're on a tall screen the dialog ends up being too tall:

too-tall


my monitors are 2560x1440, the viewport in the browser in this screenshot is 1357px tall.

File: static/js/components/ProfileImageUploader.js [hunk 2]
```diff
@@ -24,6 +25,10 @@ const dropZone = (startPhotoEdit, setPhotoError) => (
   </Dropzone>
 );
 
+const uploaderBodyHeight = (): number => (
+  window.innerHeight / 2
```
Comment (aliceriot) [inline static/js/components/ProfileImageUploader.js]: could you import `ramda` in this file and make the return value here `R.min(500, window.innerHeight / 2`? I think that should fix the dialog height issue.

**Comment by amir-qayyum-khan** [comment]
Done @aliceriot

**Comment by aliceriot** [comment]
functionality looks good!

**Pull request merged**